### PR TITLE
Pdep py3

### PR DESCRIPTION
**Pull request by alongd**
Py3 conversion of cythonized files in rmgpy/pdep:
collision
confguration
cse
me
msc
reaction
rs

The commits aren't organized by the type of change, let me know if this makes reviewing problematic

Eventually I think we can squash it all together

### CODE REVIEW

**Comment by alongd** [comment]
Most of the changes are internal variable renaming. There were just a few print and xrange statements

**Comment by mliu49** [comment]
Could you undo the renaming of function arguments? I think we will do those in a separate round of changes. Right now the tests are failing because there are calls to `calculateDensityOfStates` with keyword arguments for `activeKRotor` and `activeJRotor`.

**Comment by alongd** [comment]
@mliu49, I think this PR should be ready for review now. Looking forward for your comments. Also, I plan to squash it all into just one commit, let me know if you think otherwise.

File: rmgpy/pdep/msc.pyx [hunk 1]
```diff
@@ -30,149 +30,157 @@ Contains functionality for computing pressure-dependent phenomenological
...
 cpdef applyModifiedStrongCollisionMethod(network, str efficiencyModel='default'):
-
-    cdef numpy.ndarray[numpy.int_t,ndim=1] Jlist
-    cdef numpy.ndarray[numpy.float64_t,ndim=1] Elist, collFreq, collEff, dEdown, E0, Ereac
-    cdef numpy.ndarray[numpy.float64_t,ndim=2] A, b, K, x
-    cdef numpy.ndarray[numpy.float64_t,ndim=3] densStates
-    cdef numpy.ndarray[numpy.float64_t,ndim=4] Kij, Gnj, Fim, pa
-    cdef double T, P, E, Emin, val, beta
-    cdef int Nisom, Nreac, Nprod, Ngrains, NJ
+    """a_mat method for applying the Modified Strong Collision approach for solving the master equation."""
```
Comment (mliu49) [inline rmgpy/pdep/msc.pyx]: I think `a_mat` was an unintentional replacement of `A`.

File: rmgpy/pdep/collision.pyx [hunk 2]
```diff
@@ -31,15 +31,16 @@
 This module contains classes and functions for working with collision models.
 """
 
-cimport cython
-from libc.math cimport exp, sqrt
+from libc.math cimport exp
```
Comment (mliu49) [inline rmgpy/pdep/collision.pyx]: I know I changed this, but could you move the `libc` import to after numpy? Also in configuration.pyx, cse.pyx, and msc.pyx, and reaction.pyx.

I think libc doesn't count as a part of the standard library, but I could be wrong.

**Comment by mliu49** [comment]
You can rebase (and remove the temp fix) when you get a chance. I think this will be next for merging.

**Comment by alongd** [comment]
Done!

**Pull request merged**